Branch-SDK/Branch-SDK/BranchShareLink.m

```diff
@@ -176,11 +189,10 @@ - (void) presentActivityViewControllerFromViewController:(UIViewController*_Null
         
     }
 
-    if (self.linkProperties.controlParams[BRANCH_LINK_DATA_KEY_EMAIL_SUBJECT]) {
+    NSString *emailSubject = self.linkProperties.controlParams[BRANCH_LINK_DATA_KEY_EMAIL_SUBJECT];
```

Comment:
Adding it in control param will add it to the deep link data. Do we have a way someone can  specify the subject with out adding it to the link data?